PLAYER/SetPlayerInvincible.md

```diff
@@ -5,16 +5,12 @@ ns: PLAYER
 
 ```c
 // 0x239528EACDC3E7DE 0xDFB9A2A2
-void SET_PLAYER_INVINCIBLE(Player player, BOOL toggle);
+void SET_PLAYER_INVINCIBLE(Player player, BOOL invicible);
 ```
 
-```
-Simply sets you as invincible (Health will not deplete).  
-Use 0x733A643B5B0C53C1 instead if you want Ragdoll enabled, which is equal to:  
-*(DWORD *)(playerPedAddress + 0x188) |= (1 << 9);  
-```
+Make the player impervious to all forms of damage.
 
 ## Parameters
-* **player**: 
-* **toggle**: 
+* **player**: The player index, or -1 to get the local player ped.
```

Comment:
Usage of `-1` is discouraged, usually `PlayerPedId()` is used in Lua in favor of it.